libraries/botbuilder-ai-orchestrator/src/orchestratorAdaptiveRecognizer.ts

```diff
@@ -199,10 +234,11 @@ export class OrchestratorAdaptiveRecognizer
             if (topScore < this.unknownIntentFilterScore) {
                 recognizerResult.intents.None = { score: 1.0 };
             } else {
-                // add top score
-                recognizerResult.intents[`${topScoringIntent}`] ??= {
-                    score: topScore,
-                };
+                // add all scores
+                recognizerResult.intents = results.reduce(function(intents, result){
```

Comment:
Code formatting (run `yarn lint --fix` inside the `botbuilder-ai-orchestrator` directory.